lib/sales_analyst.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'sales_engine'
+require'pry'
+
+class SalesAnalyst
+  attr_reader :se
+
+  def initialize(se)
+    @se = se
+  end
+
+  def average_items_per_merchant
```

Comment:
Another way to build def average_items_per_merchant is: 

number_items = se.items.items.count
number_merchants = se.merchants.merchants.count
average = number_items.to_f / number_merchants
return average.round(2)

PRO: Cuts test time in half.
CON: No array with merchant item count for later use.